lua/entities/gmod_wire_expression2/core/entity.lua

```diff
@@ -19,6 +19,19 @@ local getOwner     = E2Lib.getOwner
 local isOwner      = E2Lib.isOwner
 local newE2Table   = E2Lib.newE2Table
 
+local canProperty = WireLib.CanProperty
+local canEditVariable
+
+if CPPI and debug.getregistry().Entity.CPPICanEditVariable then
+	canEditVariable = function(self, ply, k, v, e)
+		return self:CPPICanEditVariable(ply, k, v, e)
```

Comment:
Can simply have this as ``WireLib.CanEditVariable = FindMetaTable("Entity").CPPICanEditVariable`` since the arguments are the same (in wirelib ofc)

LuaLS annotations would be nice. Maybe I should have it be a rule in the developer style guide that any future WireLib/E2Lib functions need annotations